src/Platform/Microsoft.Testing.Platform.MSBuild/Tasks/TestingPlatformAutoRegisteredExtensions.cs

```diff
@@ -41,6 +41,9 @@ internal TestingPlatformAutoRegisteredExtensions(IFileSystem fileSystem)
     [Required]
     public ITaskItem[] AutoRegisteredExtensionsBuilderHook { get; set; }
 
+    [Required]
+    public string? RootNamespace { get; set; }
```

Comment:
As far as I can see it cannot be null, not sure if that's worth "handling" or not. I am also not sanitizing or anything as something else would fail anyways.